docs/user_guides/debugging/debugging_environment.rst

```diff
@@ -33,12 +33,11 @@ You will also need to create and activate conda environment with installed `numb
 Activate environment variables
 ------------------------------
 
-You need to set the following variables for debugging:
+You need to set the following variable for debugging:
```

Comment:
It's a recommendation. May be add something like:
Debugging on "no optimization" level is more stable. Local variable are not optimized out.

And provide link to Numba: http://numba.pydata.org/numba-doc/latest/user/troubleshoot.html#debugging-jit-compiled-code-with-gdb